public/lang/pl.js

```diff
@@ -145,7 +145,7 @@ export default {
 			<td>
 				<kbd>Lewy ALT</kbd> + <kbd>WASD</kbd>,
 				<br />
-				<kbd>Lewy ALT</kbd> + <kbd>strzałki</kbd>,
+				<kbd>Lewy ALT</kbd> + <kbd>Strzałki</kbd>,
```

Comment:
In the 132nd line, you can also write `Strzałki` with a capital letter.

Reply:
okay